docs/core-concepts/reusable-content-pieces/reusable-content-with-pieces.md

```diff
@@ -4,13 +4,19 @@ When you create content with the rich text editor, or images widget, that conten
 
 ## A directory of people: working with pieces
 
-Let's say you want to create a directory of people who work for a company. People are "global content": they are useful to display here and there all over the site, they aren't tied down to one page. The `apostrophe-pieces` module provides a great starting point to create many types of global content. You'll extend it to make your own `people` module. You can extend `apostrophe-pieces` many times in the same project.
+Let's say you want to create a directory of people who work for a company. People are "global content": they are useful to display in multiple places all over the site and they aren't tied down to one page. The `apostrophe-pieces` module provides a great starting point to create many types of global content. You'll extend it to make your own `people` module. You can extend `apostrophe-pieces` many times in the same project.
```

Comment:
trying my best to ignore this....

"tied down" no no no.

I have to point out, that this is just a 'content type', yeah? 'global content' is a far less popularized term here, but whatever.

Reply:
Correct. I found this confusing as well.